homeassistant/components/jewish_calendar/binary_sensor.py

```diff
@@ -55,9 +62,9 @@ def __init__(self, data, description: BinarySensorEntityDescription) -> None:
     @property
     def is_on(self) -> bool:
         """Return true if sensor is on."""
-        return self._get_zmanim().issur_melacha_in_effect
+        return self._get_zmanim().issur_melacha_in_effect  # type: ignore[no-any-return]
```

Comment:
please do a cast() instead.
```suggestion
        return cast(bool, self._get_zmanim().issur_melacha_in_effect)
```